fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -195,16 +206,115 @@ countries:
+                    var_names:
+                        obs: spi
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: *raincm
+                    lower_is_worse: yes
+                    format: number3
+                subseas-dry-tercile-snov-l22-djf:
+                    label: November biweekly forecast, 3wk lead
+                    description: Biweekly rainfall forecast issued in early November for the two week period starting two weeks after the issue date. Probability of the driest tercile.
+                    path: madagascar/subseas-dry-tercile-snov-l22-djf.zarr
+                    var_names:
+                        obs: prob
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: *raincm
+                    lower_is_worse: no
+                obs-subseas-rainfall:
+                    label: Observed subseasonal rainfall
+                    description: Observed rainfall during subseasonal forecast target period (mm)
+                    path: madagascar/obs-subseas-rainfall.zarr
+                    var_names:
+                        obs: aprod
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: *raincm
+                    lower_is_worse: yes
+                onset:
+                    label: Onset
+                    description: Onset of rainy season (day of year)
+                    path: madagascar/onset.zarr
+                    var_names:
+                        obs: onset
+                        time: time
+                    colormap: *raincm
+                    lower_is_worse: no
+                    format: number0
+                lors:
+                    label: Duration
+                    description: Length of rainy season (days)
+                    path: madagascar/onset.zarr
+                    var_names:
+                        obs: LORS
+                        time: time
+                    colormap: *raincm
+                    lower_is_worse: yes
+                    format: number0
                 enso_state: *enso_ds
                 bad-years:
-                    label: Reported bad years
-                    path: madagascar/bad-years.zarr
+                    label: Drought humanitarian ranking
+                    description: Drought ranking of expert committee humanitarian data review
+                    path: madagascar/bad-years-djf.zarr
                     var_names:
-                        obs: bad
+                        obs: rank
                         time: T
                     colormap: *raincm
-                    lower_is_worse: no
-                    format: bad
+                    lower_is_worse: yes
+                    format: number0
+                bad-years-new:
+                    label: Ranked bad years
```

Comment:
I assume they still want this column to be called "Drought humanitarian ranking." Let's call the old one v2 and the new one v3, so we don't have to change names when `new` is no longer new.

Reply:
Makes sense